test.md

```diff
@@ -0,0 +1 @@
+just for test
```

Comment:
remove this file and re-commit, otherwise your PR won't be merged.

Reply:
I have deleted the file but I can not start a new pull request, I am not sure if this works now

Reply:
It works, you are all set.